### PR TITLE
Fix the default SPICE listening address as a workaround of the VM autostart failure

### PR DESCRIPTION
**Pull request by fybmain**
Resolves the issue #1090

### CODE REVIEW

**Comment by skobyda** [comment]
Thanks. So far, this looks good. I triggered test, let's see if they pass

File: src/scripts/install_machine.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def get_graphics_capabilies(connection):
 
 def prepare_graphics_params(connection):
     graphics_config = {
-        'spice': {'listen': '127.0.0.1'},
+        'spice': {'listen': '::1'},
         'vnc': {'listen': '127.0.0.1'}
```
Comment (skobyda) [inline src/scripts/install_machine.py:50]: Could you add a comment to the bug, so in the future we know why we use ipv6 for spice

```suggestion
        # https://bugs.launchpad.net/ubuntu/+source/qemu/+bug/1492621
        'vnc': {'listen': '127.0.0.1'}
```

**Review by skobyda — CHANGES_REQUESTED** [review]

File: src/scripts/install_machine.py [hunk 2]
```diff
@@ -46,7 +46,7 @@ def get_graphics_capabilies(connection):
 
 def prepare_graphics_params(connection):
     graphics_config = {
-        'spice': {'listen': '127.0.0.1'},
+        'spice': {'listen': '::1'},
```
Comment (skobyda) [inline src/scripts/install_machine.py:49]: Could you add a comment to the bug, so in the future we know why we use ipv6 for spice

```suggestion
        # Use ipv6 as workaround for https://bugs.launchpad.net/ubuntu/+source/qemu/+bug/1492621
        'spice': {'listen': '::1'},
```

**Review by skobyda — CHANGES_REQUESTED** [review]

File: src/scripts/install_machine.py (hunk 2 above)
Comment (fybmain) [reply src/scripts/install_machine.py:49]: Wow, thank you!

**Review by skobyda — APPROVED** [review]
LGTM, thanks

**Pull request merged**